src/Templates/src/Microsoft.Maui.Templates.csproj

```diff
@@ -1,5 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <UsingTask TaskName="ReplaceFileText" TaskFactory="RoslynCodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+    <ParameterGroup>
+      <InputFilename ParameterType="System.String" Required="true" />
+      <OutputFilename ParameterType="System.String" Required="true" />
+      <MatchExpression ParameterType="System.String" Required="true" />
+      <ReplacementText ParameterType="System.String" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System" />
+      <Using Namespace="System.IO" />
+      <Using Namespace="System.Text.RegularExpressions" />
+      <Code Type="Fragment" Language="cs">
+        <![CDATA[
+            File.WriteAllText(
+                OutputFilename,
+                Regex.Replace(File.ReadAllText(InputFilename), MatchExpression, ReplacementText)
+                );
+            Console.WriteLine("Saved modified '{0}' to {1}'.", InputFilename, OutputFilename);
+          ]]>
+      </Code>
+    </Task>
+  </UsingTask>
+
```

Comment:
Does this work on .NET Core MSBuild?

Reply:
I believe so. It seemed to work on my Mac, which I think uses that?

Reply:
`RoslynCodeTaskFactory`'s work on .NET Core. `CodeTaskFactory` don't.

https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-roslyncodetaskfactory?view=vs-2022